packages/events/global.d.ts

```diff
@@ -1,47 +1,50 @@
-declare namespace GlobalMixins
-{
-    type FederatedEventEmitterTypes = import('@pixi/events').FederatedEventEmitterTypes;
-    type FederatedEventTarget = import('@pixi/events').FederatedEventTarget;
-    type IFederatedDisplayObject = import('@pixi/events').IFederatedDisplayObject;
```

Comment:
this potentially could be a breaking change. basically that type is just added to a global scope with the same name as it is declared in local, but I don't know how it could be solved with typescript syntax (after bundling it becomes `type A = A` which is an invalid syntax). but considering that you should be able to use these types via exports (or `PIXI`) it shouldn't be a problem?